Formula/liquidctl.rb

```diff
@@ -2,11 +2,12 @@ class Liquidctl < Formula
   include Language::Python::Virtualenv
 
   desc "Cross-platform tool and drivers for liquid coolers and other devices"
-  homepage "https://github.com/jonasmalacofilho/liquidctl"
+  homepage "https://github.com/liquidctl/liquidctl"
   url "https://files.pythonhosted.org/packages/cb/53/6edf9da254d2e80580b116c45a7c50edaf055917bf6d771185a5adf52d2a/liquidctl-1.7.1.tar.gz"
   sha256 "10f650b9486ddac184330940550433685ae0abc70b66fe92d994042491aab356"
   license "GPL-3.0-or-later"
-  head "https://github.com/jonasmalacofilho/liquidctl.git"
+  revision 1
```

Comment:
We don't need a revision bump here, this doesn't affect any bottles.

Reply:
That makes sense; amended and force-pushed.  Thanks for the review.